src/lib/serviceRoutines/getContextEntitiesByEntityId.cpp

```diff
@@ -49,15 +64,23 @@ std::string getContextEntitiesByEntityId
   ParseData*                 parseDataP
 )
 {
-  std::string                          entityId = (compV[0] == "v1")? compV[3] : compV[2];
-  std::string                          answer;
-  DiscoverContextAvailabilityResponse  response;
+  std::string  entityId = (compV[0] == "v1")? compV[3] : compV[2];
+  std::string  answer;
+
+  //
+  // Fill in parseDataP->dcar.res to pass to postDiscoverContextAvailability
+  //
+  EntityId                             eId(entityId, "", "");
+  std::vector<std::string>             attributeV;
+  Restriction                          restriction;
+
+  parseDataP->dcar.res.fill(eId, attributeV, restriction);
```

Comment:
One though...

Should the method name give a hint about its relation with conv ops mapping? `fill()` seems to be very generic... maybe `convFill()`. Or maybe you prefer to have `fill()` actually because is generic :)

Mark this as NTC if you think it is not a good idea to change the name. Appart from that, the idea is nice... a good use case of OOP :)